kratos/python/add_processes_to_python.cpp

```diff
@@ -539,7 +539,17 @@ void  AddProcessesToPython(pybind11::module& m)
         ;
 
     py::class_<ApplyPeriodicConditionProcess, ApplyPeriodicConditionProcess::Pointer, Process>(m,"ApplyPeriodicConditionProcess")
-            .def(py::init<ModelPart&,ModelPart&, Parameters>())
+    .def(py::init<ModelPart&,ModelPart&, Parameters>())
+    ;
+
+    py::class_<FindIntersectedGeometricalObjectsProcess<Element>, FindIntersectedGeometricalObjectsProcess<Element>::Pointer, Process>(m,"FindIntersectedElementsProcess")
```

Comment:
```suggestion
    py::class_<FindIntersectedGeometricalObjectsProcess<Element>, FindIntersectedGeometricalObjectsProcess<Element>::Pointer, Process>(m,"ElementBasedFindIntersectedGeometricalObjectsProcess")
```

Reply:
Is there any place in where we directly use this in Python? If there isn't I wouldn't export it.